packages/labs/eleventy-plugin-lit/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2021 Google LLC. All rights reserved.
```

Comment:
is the year meant to be for when this file first appears in the repo? should it say 2022?

Reply:
It first appeared in 2021, since that's when Kevin sent this PR out.